tsconfig.json

```diff
@@ -1,9 +1,15 @@
-/* Dev configuration */
 {
-  "extends": "./tsconfig.base.json",
-  "compilerOptions": {
-    "esModuleInterop": true, // Required fto remove warnings or jest test
-  },
-  /* During dev, src are included so they are also modified during refactorings. */
-  "include": ["src/**/*.ts", "test/**/*.ts"],
+    "compilerOptions": {
```

Comment:
I'd recommend extending from a `@tsconfig/*` package, e.g. https://github.com/tsconfig/bases/#node-lts-tsconfigjson and only add necessary changes.

Reply:
We could go with @tsconfig/node-lts or with @tsconfig/node16 ... @mfucci @turon What do you think?